handbook/engineering/code-intelligence/index.md

```diff
@@ -12,6 +12,20 @@ The Code Intelligence team builds tools and services that provide contextual inf
 
 - Robust, extensible, and scalable infrastructure to index code across all languages, keep those indexes up-to-date, and efficiently resolve code intelligence queries against all indexed code.
 
+## History
+
+The origin of this team is the core of Sourcegraph, the first iteration was [srclib](https://github.com/sourcegraph/srclib) a code analysis tool developed by our co-founders [Beyang](https://github.com/beyang) and [Quinn](https://github.com/sqs) in 2015.
```

Comment:
The first idea here seems unfinished and unrelated to the second.

Reply:
What do you mean by unfinished? It's the genesis in short and sweet format - I don't intend for it to be comprehensive.

Reply:
> The origin of this team is the core of Sourcegraph

It's not clear to me how this should be interpreted. Does this mean src lib was the core of Sourcegraph and also the origin for code intelligence? Or that our team is the core of Sourcegraph?

Reply:
Oh I see, you mean "our team" as in the people here today were part of original devs?

Reply:
Yeah, I think it's throwing me at "the first iteration [of what?] ...".

Reply:
Updated

Reply:
Thanks! This makes a lot more sense to me.